packages/database/src/core/modelClasses/SyncSession.js

```diff
@@ -6,26 +6,6 @@ import { RECORDS } from '../records';
 
 export class SyncSessionRecord extends DatabaseRecord {
   static databaseRecord = RECORDS.SYNC_SESSION;
-
-  /**
-   * Mark the sync session as started at a given tick
-   * @param {*} tick 
-   * @returns 
-   */
-  async markAsStartedAt(tick) {
-    return this.model.update({ id: this.id }, { started_at_tick: tick });
-  }
-
-  /**
-   * Mark the sync session as errored with a given error
-   * @param {*} error 
-   */
-  async markErrored(error) {
-    const errors = this.errors || [];
-    this.errors = [...errors, error];
-    this.completedAt = new Date();
-    await this.save();
-  }
```

Comment:
I'm removing all of these utils in SessionRecord. This is because DatabaseRecord.save() actually set all the values when generating update statement, including null values. And with a few asynchronous actions during sync sessions (eg: initiate snapshot and then polling,...), that sometimes cause some values to be overwritten by null values.